tests/unittest/_torch/modules/test_dwdp_fixup_moe_backends.py

```
@@ -0,0 +1,459 @@
+        return gate
+
+    def test_missing_bias_is_noop(self):
+        gate = nn.Module()
+        comm = _make_mock_comm([torch.zeros(2)])
+        _allgather_e_score_correction_bias(
+            gate, layer_idx=3, dwdp_rank=0, dwdp_size=4, num_experts_total=16, comm=comm
+        )
+        comm.allgather.assert_not_called()
+        self.assertFalse(
+            hasattr(gate, "e_score_correction_bias")
+            and getattr(gate, "e_score_correction_bias") is not None
```

Ruff (0.15.14)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)